.github/workflows/e2e.yml

```diff
@@ -84,60 +87,17 @@ jobs:
         run: make -j2 docker runner tests
         if: "env.GIT_DIFF != ''"
 
-      - name: Run CI dashcore testnet
+      - name: Run e2e test
         working-directory: test/e2e
-        run: ./run-multiple.sh networks/dashcore.toml
+        run: ./build/runner -f networks/${{matrix.manifest}}.toml
         if: "env.GIT_DIFF != ''"
 
-      - name: Emit dashcore logs on failure
+      - name: Emit logs on failure
         if: ${{ failure() }}
         working-directory: test/e2e
-        run: ./build/runner -f networks/dashcore.toml logs
+        run: ./build/runner -f networks/${{matrix.manifest}}.toml logs
 
-  e2e-test-rotate:
-    runs-on: ubuntu-latest
-    needs: bls-signatures
-    timeout-minutes: 35
-    env:
-      FULLNODE_PUBKEY_KEEP: false
-    steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: '1.17'
-      - uses: actions/checkout@v2.3.4
-        with:
-          submodules: true
-      - name: Get BLS library revision
-        id: bls-revision
-        run: |
-          echo "::set-output name=hash::$(git --git-dir=third_party/bls-signatures/src/.git rev-parse HEAD)"
-        shell: bash
-      - uses: technote-space/get-diff-action@v5.0.2
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
-      - uses: actions/cache@v2
-        with:
-          path: ~/bls-cache
-          key: ${{ runner.os }}-bls-${{ steps.bls-revision.outputs.hash }}
-      - name: Install BLS library
-        run: |
-          sudo cp -vr ~/bls-cache/include/* /usr/local/include/
-          sudo cp -vr ~/bls-cache/libchiabls.a /usr/local/lib/
-      - name: Build
+      - name: Cleanup e2e network configuration
```

Comment:
maybe it should be a [post](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspost) action?